### PR TITLE
Update the AWS Terraform provider version to 1.60.0

### PR DESCRIPTION
**Pull request by cbaines**
This is the latest minor version before major version 2.

From 1.40.0 in most cases, apart from app-prometheus and
infra-datagovuk-organogram-bucket which were on 1.14.0.

### CODE REVIEW

**Comment by afda16** [comment]
https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1420-october-31-2018 -- I think vpc_id in private route53 zone resources, such as https://github.com/alphagov/govuk-aws/blob/master/terraform/projects/infra-root-dns-zones/main.tf#L83, may need to be updated

**Comment by afda16** [comment]
Apart from the aws_route53_zone changes, I think the rest of changes in the provider should be compatible

**Comment by cbaines** [comment]
> https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1420-october-31-2018 -- I think vpc_id in private route53 zone resources, such as https://github.com/alphagov/govuk-aws/blob/master/terraform/projects/infra-root-dns-zones/main.tf#L83, may need to be updated

Maybe I missed some, but this was actually the only example of `vpc_id` being used in a `aws_route53_zone` resource. I've now pushed a commit to use the `vpc { }` block though.

File: terraform/projects/infra-mirror-bucket/main.tf [hunk 1]
```diff
@@ -57,13 +57,13 @@ terraform {
...
-  version = "1.40.0"
+  version = "1.60.0"
 }
 
 provider "aws" {
-  region  = "${var.aws_replica_region}"
-  alias   = "aws_replica"
-  version = "1.40.0"
+  region  = "${var.aws_backup_region}"
+  alias   = "aws_secondary"
```
Comment (afda16) [inline terraform/projects/infra-mirror-bucket/main.tf]: If you update the alias, I think you need to update the line where it's referenced

File: terraform/projects/infra-mirror-bucket/main.tf [hunk 2]
```diff
@@ -57,13 +57,13 @@ terraform {
...
   region  = "${var.aws_region}"
-  version = "1.40.0"
+  version = "1.60.0"
 }
 
 provider "aws" {
-  region  = "${var.aws_replica_region}"
-  alias   = "aws_replica"
-  version = "1.40.0"
+  region  = "${var.aws_backup_region}"
```
Comment (afda16) [inline terraform/projects/infra-mirror-bucket/main.tf]: Does this variable exist in the file?

File: terraform/projects/infra-mirror-bucket/main.tf (hunk 1 above)
Comment (cbaines) [reply terraform/projects/infra-mirror-bucket/main.tf]: I believe this change was me just getting a rebase wrong, I've changed the commit to just change the version now. Good spot though!

File: terraform/projects/infra-mirror-bucket/main.tf (hunk 2 above)
Comment (cbaines) [reply terraform/projects/infra-mirror-bucket/main.tf]: Likewise, I believe this change was me just getting a rebase wrong, I've changed the commit to just change the version now.

**Pull request merged**